### PR TITLE
add proportion_explained to ca, fixes #1345

### LINKED ISSUE #1345: BUG: CA missing proportion_explained
Looks like correspondence analysis is missing the `proportion_explained` variable [here](https://github.com/biocore/scikit-bio/blob/master/skbio/stats/ordination/_correspondence_analysis.py#L188).

There should be 

`proportion_explained=eigvals / eigvals.sum()`

All of the other ordination methods have this.

### PR DESCRIPTION
**Pull request by nreeve17**
Please complete the following checklist:

* [x] I have read the guidelines in [CONTRIBUTING.md](https://github.com/biocore/scikit-bio/blob/master/CONTRIBUTING.md).

* [x] I have documented all public-facing changes in [CHANGELOG.md](https://github.com/biocore/scikit-bio/blob/master/CHANGELOG.md).

* [ ] **This pull request includes code, documentation, or other content derived from external source(s).** If this is the case, ensure the external source's license is compatible with scikit-bio's [license](https://github.com/biocore/scikit-bio/blob/master/COPYING.txt). Include the license in the `licenses` directory and add a comment in the code giving proper attribution. Ensure any other requirements set forth by the license and/or author are satisfied. **It is your responsibility to disclose code, documentation, or other content derived from external source(s).** If you have questions about whether something can be included in the project or how to give proper attribution, include those questions in your pull request and a reviewer will assist you.

* [x] **This pull request does not include code, documentation, or other content derived from external source(s).**

**Note:** [REVIEWING.md](https://github.com/biocore/scikit-bio/blob/master/REVIEWING.md) may also be helpful to see some of the things code reviewers will be verifying when reviewing your pull request.

### CODE REVIEW

**Comment by wasade** [comment]
@nreeve17, could you please clarify the check boxes? There are conflicting responses. The checkbox indicating this *does* have externally derived content is checked, as well as the one indicating that this *does not* have externally derived content.

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,8 @@
 ### Bug fixes
 * The `include_self` parameter was not being honored in `skbio.TreeNode.tips`. The scope of this bug was that if `TreeNode.tips` was called on a tip, it would always result in an empty `list` when unrolled.
 
+* In `ca`, `proportion_explained` is inclued in the returned `OrdinationResults` object. ([#1345](https://github.com/biocore/scikit-bio/issues/1345))
```
Comment (wasade) [inline CHANGELOG.md]: Can the module path to `ca` be added? eg `skbio.stats.ordination.ca`

"is included" -> "was missing"

**Comment by wasade** [comment]
It looks like there are some linting [failures](https://travis-ci.org/biocore/scikit-bio/jobs/230963236#L632-L634).

**Comment by nreeve17** [comment]
Sorry about that, I misunderstood the third box. The third box should not
have been checked, there is no externally derived content in my edit. How
do I go about fixing that, do I just uncheck the box?

**Comment by wasade** [comment]
Ya :)

**Comment by ElDeveloper** [comment]
Just a note, seems like there's quite a few unrelated failures, most of them having to do something with positional metadata.

**Comment by RNAer** [comment]
fixed via #1553

**Pull request closed without merging**